all/pom.xml

```diff
@@ -163,11 +163,11 @@
               <promoteTransitiveDependencies>true</promoteTransitiveDependencies>
               <filters>
                 <filter>
-                  <artifact>org.bouncycastle:bcprov-jdk15on</artifact>
+                  <artifact>*:*</artifact>
```

Comment:
Couldn't we keep it as org.bouncycastle:*

Reply:
We can keep the smaller scope, but I think we never want to include the signature files. So upgrade of a library which suddenly starts to sign its JARs wouldn't require the filter change.

@coheigea So do you think we should have the smaller scope and filter the libraries with signed classes if a need comes?

Reply:
Makes me think that we have some trouble with the jar we are delivering, since M24: they are not signed anymore...

Reply:
@kwart Yeah you're right, let's leave it as it is.